introduction.md

```diff
@@ -1 +1,7 @@
 Hi I'm Matty B, my favorite Code Path is Electives
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?